libs/datamodel/core/tests/config/generators.rs

```diff
@@ -212,10 +212,47 @@ fn nice_error_for_unknown_generator_preview_feature() {
 }
 
 #[test]
-fn retain_env_var_definitions_in_generator_block() {
-    std::env::set_var("PROVIDER", "postgres");
-    std::env::set_var("OUTPUT", "~/home/prisma/");
```

Comment:
not required in old test any more?

Reply:
Indeed — this is part of a refactoring effort I started a few weeks ago to have the datamodel parser _not_ read env vars during parsing. This surfaced that weirdness with binaryTargets. There's still a bunch of code manipulating env vars in datamodel tests, but I hope we can remove most of it progressively.